link/src/main/java/com/stripe/android/link/LinkActivityContract.kt

```diff
@@ -34,6 +34,9 @@ class LinkActivityContract :
      * @param injectionParams Parameters needed to perform dependency injection.
      *                        If null, a new dependency graph will be created.
      */
+    @Deprecated(
```

Comment:
We should be able to remove all of these (or just add another `@RestrictTo`) since the parent class is already restricted.